testing/testcafe/tests/editors/lookup.ts

```diff
@@ -42,3 +49,60 @@ test('Popover should have correct vertical position (T1048128)', async (t) => {
     .expect(lookupElementBottom)
     .eql(popoverArrowTop);
 });
+
+fixture`Lookup`
+  .page(url(__dirname, '../container.html'));
+
+themes.forEach((theme) => {
+  test(`Check popup height with no found data option, theme=${theme}`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t.click(Selector(`.${LOOKUP_FIELD_CLASS}`));
+
+    await t
+      .expect(await takeScreenshot(`Lookup_with_no_found_data,theme=${theme.replace(/\./g, '-')}.png`))
+      .ok()
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  }).before(async (t) => {
+    await t.resizeWindow(300, 400);
```

Comment:
why do we need resize the window?

Reply:
The dimensions of popup with input are different in various themes. This windows size satisfies all cases.